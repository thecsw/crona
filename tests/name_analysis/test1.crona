a : int;

foo : void(b : int) {
	a = b;
}

main : int() {
	foo(10);
	write a;
	return 0;
}