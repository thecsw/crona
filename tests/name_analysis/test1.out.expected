a(int):int;
foo(int->void):void(b(int):int){
	a(int) = b(int);
}
main(->int):int(){
	foo(int->void)(10);
	write a(int);
	return 0;
}